modules/imgproc/src/drawing.cpp

```diff
@@ -2364,6 +2364,17 @@ Size getTextSize( const String& text, int fontFace, double fontScale, int thickn
     return size;
 }
 
+double getFontScaleFromHeight(const int fontFace, const int pixelHeight, const int thickness)
+{
+    // By https://stackoverflow.com/a/27898487/1531708
```

Comment:
I don't think this link is helpfull here since stakoverflow is pointing to opencv:
> I have found out that the cv::getTextSize in opencv looks as follows

Reply:
The link points to the answer containing the code for ``getFontScaleFromHeight()``

Reply:
Yep, but this code is pretty similar to getTextSize code. So there is no need to external links.